Python/Tests/Core/DebugReplEvaluatorTests.cs

```diff
@@ -51,7 +51,7 @@ public class DebugReplEvaluatorTests {
 
         internal virtual PythonVersion Version {
             get {
-                return PythonPaths.Python26 ?? PythonPaths.Python26_x64;
+                return PythonPaths.Python27 ?? PythonPaths.Python27_x64;
```

Comment:
As this is a virtual property, presumably there's an override somewhere for running the 2.7 tests that you'll need to remove?

Reply:
This class has been turned into abstract, so it will not run the 2.7 tests which continue to run in "DebugReplEvaluatorTests27". This is consistent with the other code (such as "DebuggerTests") where the base class is abstract so no tests will ever be run on it. The "correct" way of doing this would be to convert this to an abstract property, but I don't want to break away from the norm for this one case. Perhaps I'll change it when i go through all the broken tests to fix them.

Reply:
This class has been turned into abstract, so it will not run the 2.7 tests which continue to run in "DebugReplEvaluatorTests27". This is consistent with the other code (such as "DebuggerTests") where the base class is abstract so no tests will ever be run on it. The "correct" way of doing this would be to convert this to an abstract property, but I don't want to break away from the norm for this one case. Perhaps I'll change it when i go through all the broken tests to fix them.